src/main/host/syscall/mman.c

```diff
@@ -165,8 +165,11 @@ static SysCallReturn _syscallhandler_mmap(SysCallHandler* sys, PluginPtr addrPtr
     }
 
     // Delegate execution of the mmap itself to the memorymanager.
```

Comment:
Could we update these delegation comments to explain why/when the memory manager could be NULL and why NATIVE is OK in that case?

Reply:
Done. (Changed to assert)